ml_perf/scripts/train.sh

```diff
@@ -0,0 +1,20 @@
+source ml_perf/scripts/common.sh
+
+
+# Set up an exit handler that stops the selfplay workers.
+function stop_selfplay {
+  /ml_perf/scripts/stop_selfplay.sh \
+    --board_size="${board_size}" \
+    --base_dir="${base_dir}" \
+}
+trap stop_selfplay EXIT
```

Comment:
Should we trap sigterm also?

Reply:
IIUC `EXIT` covers `SIGTERM` too because the handler is always run when the script exits, regardless of the reason.

Reply:
got it, thanks!